OpenRA.Game/WVec.cs

```diff
@@ -82,8 +82,8 @@ public static WVec LerpQuadratic(WVec a, WVec b, WAngle pitch, int mul, int div)
 				return ret;
 
 			// Add an additional quadratic variation to height
-			// Uses fp to avoid integer overflow
-			var offset = (int)((float)(b - a).Length * pitch.Tan() * mul * (div - mul) / (1024 * div * div));
+			// Uses long to avoid integer overflow
```

Comment:
This isn't a hot path, so we could probably get away with using `decimal` here (there's another similar line somewhere using long that could also change) and remove the risk of overflow completely.